Stock.Api/Controllers/ProviderController.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Stock.Api.DTOs;
+using Stock.Api.Extensions;
+using Stock.AppService.Services;
+using Stock.Model.Entities;
+
+namespace Stock.Api.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/provider")]
+    [ApiController]
+    public class ProviderController : ControllerBase
+    {
+        private ProviderService service;
+        private readonly IMapper mapper;
+
+        public ProviderController(ProviderService service, IMapper mapper)
```

Comment:
Para cada método del controller, agregar los tags arriba para que swagger genera la documentación
ver controllers existentes como ejemplo

Reply:
Me da el siguiente error al compilar:

PS D:\Hexacta\HxLabs\HexactaLabs-NetCore_React-Initial> cd stock.web
PS D:\Hexacta\HxLabs\HexactaLabs-NetCore_React-Initial\stock.web> cd client-app
PS D:\Hexacta\HxLabs\HexactaLabs-NetCore_React-Initial\stock.web\client-app> cd src
PS D:\Hexacta\HxLabs\HexactaLabs-NetCore_React-Initial\stock.web\client-app\src> npm start

> client-app@0.1.0 start D:\Hexacta\HxLabs\HexactaLabs-NetCore_React-Initial\stock.web\client-app
Failed to compile.

./node_modules/react-table/react-table.css (./node_modules/css-loader/dist/cjs.js??ref--6-oneOf-3-1!./node_modules/postcss-loader/src??postcss!./node_modules/react-table/react-table.css)
Error: No valid exports main found for 'D:\Hexacta\HxLabs\HexactaLabs-NetCore_React-Initial\Stock.Web\client-app\node_modules\colorette'